app/jobs/maintenance_tasks/task_job.rb

```diff
@@ -97,7 +97,12 @@ def after_perform
     def on_error(error)
       @ticker.persist if defined?(@ticker)
       @run.persist_error(error)
-      MaintenanceTasks.error_handler.call(error)
+      task_context = {
+        task_name: @run.task_name,
+        started_at: @run.started_at,
+        ended_at: @run.ended_at,
+      }
+      MaintenanceTasks.error_handler.call(task_context, error)
```

Comment:
We need to keep backwards compatibility for existing users who have already configured their error handler. I think we can add an arity check here.

Reply:
Thanks, I forgot that would be an issue ❤️